InteractionBase/src/vis/Menu.cpp

```diff
@@ -140,12 +140,13 @@ void Menu::updateGeometry(int availableWidth, int availableHeight)
 
 bool Menu::sceneEventFilter(QGraphicsItem* watched, QEvent* event)
 {
-	//If we are in focus mode, and not F2 is pressed, we don't handle any events here.
+	//If we are in focus mode, and not F2 or Return is pressed, we don't handle any events here.
 	if (inFocusMode_)
 	{
 		bool isFocusModeKey = false;
 		if (event->type() == QEvent::KeyPress)
-			if (static_cast<QKeyEvent*>(event)->key() == Qt::Key_F2)
+			if (static_cast<QKeyEvent*>(event)->key() == Qt::Key_F2
+					|| static_cast<QKeyEvent*>(event)->key() == Qt::Key_Return)
```

Comment:
When handling return you should always check for either one of two keys `Qt::Key_Return` and `Qt::Key_Enter`